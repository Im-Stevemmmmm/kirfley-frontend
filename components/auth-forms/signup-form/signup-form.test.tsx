it.todo("It should wokr");
