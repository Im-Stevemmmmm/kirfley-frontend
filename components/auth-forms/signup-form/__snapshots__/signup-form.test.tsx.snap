// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <form
    class="form"
  >
    <h1
      class="form__title"
    >
      Sign Up
    </h1>
    <div
      class="dobContainer"
    >
      <input
        class="form__input dobContainer__month"
        id="month"
        name="month"
        placeholder="Month"
        type="number"
        value="0"
      />
      <input
        class="form__input dobContainer__day"
        id="day"
        name="day"
        placeholder="Day"
        type="number"
        value="0"
      />
      <input
        class="form__input dobContainer__year"
        id="year"
        name="year"
        placeholder="Year"
        type="number"
        value="0"
      />
    </div>
    <label
      class="form__label"
      for="username"
    >
      Username
    </label>
    <input
      class="form__input"
      id="username"
      name="username"
      placeholder="Atleast 4 characters long"
      type="text"
      value=""
    />
    <label
      class="form__label"
      for="email"
    >
      Email
    </label>
    <input
      class="form__input"
      id="email"
      name="email"
      placeholder="example@email.com"
      type="text"
      value=""
    />
    <label
      class="form__label"
      for="password"
    >
      Password
    </label>
    <input
      class="form__input"
      id="password"
      name="password"
      placeholder="Atleast 4 characters long"
      type="password"
      value=""
    />
    <label
      class="form__label"
      for="confirmPassword"
    >
      Confirm Password
    </label>
    <input
      class="form__input"
      id="confirmPassword"
      name="confirmPassword"
      type="password"
      value=""
    />
    <div
      class="buttonContainer"
    >
      <button
        class="form__button standard"
        type="submit"
      >
        Sign Up
      </button>
      <p
        class="buttonContainer__text"
      >
        or
      </p>
      <button
        class="form__button standard"
        type="button"
      >
        Back to log in
      </button>
    </div>
  </form>
</div>
`;
